didcomm_v2/message_dispatch.py

```diff
@@ -12,9 +12,10 @@
 from protocols.action_menu import process_action_menu_message
 from protocols.shorten_url import process_shorten_url_message
 from db_utils import create_connection, get_connection, update_connection
+
+from protocols.presentation_exchange import process_presentation_exchange_message
 import os
-if "PRISM_ISSUER" in os.environ and os.environ["PRISM_ISSUER"]==1:
-    from protocols.issue_credential import process_issue_credential_message
+from protocols.issue_credential import process_issue_credential_message
```

Comment:
we need to fix the not working `if` for the case that someone only needs the Mediator part. We can do it an issue later